src/twisted/conch/test/test_forwarding.py

```diff
@@ -65,17 +40,13 @@ def test_channelOpenHostnameRequests(self):
         is resolved using HostnameEndpoint's resolver.
         """
         sut = forwarding.SSHConnectForwardingChannel(
-            hostport=('fwd.example.org', 1234))
+            hostport=(b'fwd.example.org', 1234))
```

Comment:
Is this making it satisfy the current interface of `SSHConnectForwardingChannel`? Or will it not accept unicode now (on py3, if it's even ported)?